heralding/capabilities/http.py

```diff
@@ -66,6 +73,11 @@ def do_GET(self):
             self.wfile.write(headers_bytes)
             self.wfile.write(b'not authenticated')
 
+        logger.debug(self.headers['User-Agent'])
```

Comment:
Did you study how we log data in Heralding? There are two "types" of logging, one if stuff regarding Heralding itself the other is the stuff we log about the clients that try to authenticate. Logging the User agent would be the latter, hence this is not the proper way to do this. Try studying the code in the Reporting directory.